functions/shorten.js

```diff
@@ -0,0 +1,39 @@
+const Discord = require("discord.js");
```

Comment:
you can remove this line since it's not declared anywhere

Reply:
cool

Reply:
~~doubt you used GitHub before~~ umm you can remove that line since it's not necessary to be there

Reply:
yeah i don't use github a lot this is my first contribute :3 

sry for forget a lot of things i'm not used to it :)

Reply:
> you can remove this line since it's not declared anywhere

waiting for this...